licenses/github.com/xeipuuv/gojsonpointer/NONE

```diff
@@ -0,0 +1 @@
+NO LICENSE FOUND
```

Comment:
is this a problem for us?

Reply:
It is. We have a manual whitelist process. This does have a license its just not found by our linter due to a weird name - which is why this PR fails until common files updates, where i updated the whitelist